google/cloud/forseti/services/utils.py

```diff
@@ -168,6 +168,9 @@ def to_full_resource_name(full_parent_name, resource_type_name):
     Returns:
         str: full_resource_name of the child
     """
+    # Strip out the fake composite root parent from the full resource name.
+    if full_parent_name == 'composite_root/root/':
```

Comment:
Just to clarify, should this be 'composite_root/root/' or 'composite_root/'?

Reply:
It is correct, the type is composite_root and the key is root.